app/bundles/LeadBundle/Entity/TagRepository.php

```diff
@@ -61,7 +61,7 @@ public function getTagsByName($tags)
         array_walk($tags, create_function('&$val', 'if (strpos($val, "-") === 0) $val = substr($val, 1);'));
         $qb = $this->_em->createQueryBuilder()
             ->select('t')
-            ->from('MauticLeadBundle:Tag', 't', 't.tag');
+            ->from(Tag::class, 't', 't.tag');
```

Comment:
Why not to use `$this->createQueryBuilder('t', 't.tag')`?

Reply:
If such syntax would work, I'll change it.

Reply:
There is no reason why this would not work.